blocks/header/header.js

```diff
@@ -133,11 +133,6 @@ class Gnav {
 
     const brand = brandBlock.querySelector('a');
     brand.classList.add('gnav-brand', 'link-highlight-colorful-effect-hover-wrapper');
-
-    // accessibility
-    const brandAriaLabel = brandBlock.textContent ? brandBlock.textContent.trim() : 'Adobe Franklin';
-    brand.setAttribute('aria-label', brandAriaLabel);
```

Comment:
I think you would only ever need to add an `aria-label` if the element had no text content...?

Reply:
would it still be needed if there is an `alt-text` or `svg` `title`?

Reply:
Good point. No, I don't think it would be.